packages/web-client/app/components/card-pay/deposit-workflow/index.ts

```diff
@@ -186,8 +192,8 @@ class DepositWorkflowComponent extends Component {
     this.workflow = new DepositWorkflow(getOwner(this));
   }
 
-  @action cancelWorkflow() {
-    this.workflow.cancel();
+  @action onDisconnect() {
+    this.workflow.cancel('DISCONNECTED');
```

Comment:
Consider making 'DISCONNECTED' a constant

Reply:
Will make `DISCONNECTED` and `INSUFFICIENT_FUNDS` constants in workflow-config.

Reply:
Added these constants as entries within a typescript const type:

https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums